frontend/simple/views/components/Menu/MenuParent.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="c-menu">
+    <slot></slot>
+  </div>
+</template>
+<style lang="scss" scoped>
+.c-menu {
+  position: relative;
+}
+</style>
+<script>
+export default {
+  name: 'MenuSelect',
+  data () {
+    return {
+      Menu: {
+        isActive: this.isActive,
+        handleSelect: this.handleSelect,
+        handleTrigger: this.handleTrigger,
+        closeMenu: this.closeMenu
+      }
+    }
+  },
+  provide () {
+    return {
+      Menu: this.Menu
+    }
+  },
+  watch: {
+    isActive: {
+      immediate: true,
+      handler (isActive) {
+        this.Menu.isActive = isActive
+      }
+    }
+  },
+  computed: {
+    isActive () {
+      return this.Menu.isActive
+    }
+  },
+  methods: {
+    handleTrigger () {
+      this.Menu.isActive = true
+      this.$emit('open')
+    },
+    handleSelect (itemId) {
+      this.closeMenu()
+      this.$emit('select', itemId)
```

Comment:
- What are these two events for?
- What is listening to them?
- Should they use SBP events instead?

Reply:
Components that use `<menu-parent>` can listen for those events.

ATM `GroupsList.vue` is listening for `select` and once `Activity.vue` works correctly it will do it too (on a next PR).

I didn't use SBP here because `this.$emit` / `@select` is a specific Vue feature to handle props callbacks.

```
<menu-parent @select="handleMenuSelect">
```

It would be the same as doing the following:


```js
handleSelect (itemId) {
      this.closeMenu()
      this.onSelect(itemId) // onSelect being a prop
}
```

```vue
<menu-parent onSelect="handleMenuSelect">
```

but I found `@` prop more readable.

ATM `open` prop isn't being listen anywhere, so I'll delete it and once is needed we can add it again.